rawfile/rawfile_servicer.py

```diff
@@ -298,18 +269,25 @@ def volume_provision(usage: Usage):
 
     @log_grpc_request
     def DeleteVolume(self, request, context):
-        size = 0
-        try:
-            size = scrub(volume_id=request.volume_id)
-        except CalledProcessError as exc:
-            if exc.returncode == VOLUME_IN_USE_EXIT_CODE:
-                context.abort(grpc.StatusCode.FAILED_PRECONDITION, "Volume in use")
-            else:
-                raise exc
+        img_dir = utils.rawfile.img_dir(request.volume_id)
+        if not img_dir.exists():
+            return 0
+
+        img_file = utils.rawfile.img_file(request.volume_id)
```

Comment:
any reason to move this here? Seemed a bit tidier in a separate function

Reply:
Done.